.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - RUN_PYLINT="true"
     - RUN_MYPY="true"
     - BUILD_DOCS="true"
+    - CHECK_DOCS="true"
```

Comment:
I'd put this on the same line as `BUILD_DOCS`, so you both build the docs and check the docs in the same build.  We're allowed four concurrent builds with travis, but five will slow things down considerably.